packages/uploadthing/test/upload-builder.test.ts

```diff
@@ -15,6 +15,9 @@ it("typeerrors for invalid input", () => {
   // @ts-expect-error - invalid size format
   f({ image: { maxFileSize: "1gb" } });
 
+  // TODO: add test for minFileCount <= maxFileCount
+  //f({ image: { minFileCount: 2, maxFileCount: 1 } });
```

Comment:
i think this will be tough to assure on the typelevel - runtime check should be enough?

Reply:
When would that check happen though?

Reply:
Ok, so not when expanding the config, actually when you try to do an upload. Got it.